gateway/handle_http.go

```diff
@@ -66,6 +69,24 @@ func (*Handle) robotsHandler(w http.ResponseWriter, _ *http.Request) {
 	_, _ = w.Write([]byte("User-agent: * \nDisallow: / \n"))
 }
 
+// workspaceStatusHandler - handler for workspace status requests
+func (gw *Handle) workspaceStatusHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		workspaceID := chi.URLParam(r, "workspaceID")
```

Comment:
Shouldn't we at least check if the workspaceID is empty and return a 400? Also, do we have a format for the workspaceID or it can be any string?